Models.py

```diff
@@ -106,7 +136,15 @@ def save(self) -> None:
             json.dump(self.data, json_file)
 
     def test(self) -> None:
-        """EXPENSIVE"""
+        """
+        A method for testing purposes. It's EXPENSIVE and should only be used for testing purposes.
+
+        Args:
+            None
+        
+        Returns:
+            None
+        """
```

Comment:
Could you change this to:

"""A method for testing purposes."""

Also, could you add a warning in the docstrings?